ValueError: Game screen not found.

https://fgojunks.max747.org/fgosccnt/results/4e74c3fe-e9ae-4381-a3ea-bbbee85f7cb6/
で発生

```
ValueError: Game screen not found.
```
とでてgame screenの取得に失敗する

![0001](https://user-images.githubusercontent.com/62515228/132959399-d43e816d-99c4-4399-bacd-891205ac715e.jpg)

```
> python ..\fgosccnt.py 0001.jpg -l debug
__main__ <fgosccnt.py-L2455> [DEBUG] filename: 0001.jpg
__main__ <fgosccnt.py-L2554> [ERROR] 0001.jpg
__main__ <fgosccnt.py-L2555> [ERROR] Game screen not found.
Traceback (most recent call last):
  File "C:\Users\fgophi\Documents\fgosc\fgosccnt\fgosccnt.py", line 2471, in get_output
    sc = ScreenShot(args, img_rgb,
  File "C:\Users\fgophi\Documents\fgosc\fgosccnt\fgosccnt.py", line 350, in __init__
    ((self.x1, self.y1), (self.x2, self.y2)) = get_coodinates(self.img_rgb_orig)
  File "C:\Users\fgophi\Documents\fgosc\fgosccnt\fgosccnt.py", line 271, in get_coodinates
    raise ValueError("Game screen not found.")
ValueError: Game screen not found.
filename,ドロ数
0001.jpg: not valid,
```

--- a/fgosccnt.py
+++ b/fgosccnt.py
@@ -265,7 +265,7 @@ def get_coodinates(img: ndarray,
     for cnt in contours:
         _, _, w, h = cv2.boundingRect(cnt)
         area = cv2.contourArea(cnt)
-        if 1.82 < w/h < 1.83 and area > height / 2 * width / 2:
+        if 1.81 < w/h < 1.83 and area > height / 2 * width / 2:
             contours2.append(cnt)
     if len(contours2) == 0:
         raise ValueError("Game screen not found.")

